.ci/travis/travis.sh

```diff
@@ -287,19 +299,21 @@ no-error-xwiki)
   mvn -e install:install-file -Dfile=xwiki-commons-pom/pom.xml \
     -DpomFile=xwiki-commons-pom/pom.xml
   mvn -e -f xwiki-commons-tools/xwiki-commons-tool-webjar-handlers/pom.xml \
-    install -Dmaven.test.skip
+    install -Dmaven.test.skip -Dcheckstyle.version=${CS_POM_VERSION}
   mvn -e -f xwiki-commons-tools/xwiki-commons-tool-xar/pom.xml \
-  install -Dmaven.test.skip
+    install -Dmaven.test.skip -Dcheckstyle.version=${CS_POM_VERSION}
```

Comment:
These version overrides were missing. CI was failing during breaking change saying it didn't understand the new properties without these in place.